packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <RemovePositioningDuringScreenshot
     id="main-pane"
-    class="flex border-gray-900 border-l-1"
+    class="flex border-gray-900"
+    :class="screenshotStore.isScreenshotting ? '' : 'border-l-1'"
```

Comment:
`RemovePositioningDuringScreenshot.vue` already has conditional logic on i`Screenshotting`:

```js
// ...

const style = computed(() => {
  if (screenshotStore.isScreenshotting) {
    return {
      left: `0px`,
      width: `100%`,
    }
  }

  return {
    width: `calc(100% - ${navbarWidth})`,
  }
})
```

Can we just add this class inside there? The idea of `<XXXDuringScreenshot>` is to keep all the screenshot-specific conditional styles in a single place, rather than it leaking out into the rest of the app.

Reply:
I didn't initially because I didn't look at all the places we are using `RemovePositioningDuringScreenshot` - maybe it's just in this though, in which case cool, I know we have a "remove classes" helper too.

But yeah, point taken about the leaking, let's add it here.